src/rabbitmanager.js

```diff
@@ -66,6 +67,66 @@ class RabbitManager {
       method: 'delete',
     });
   }
+
+  async users() {
+    return await this.request('users');
+  }
+
+  async usersWithAllTags(tags=[]) {
+    let userList = await users();
+    return _filterUsersWithAllTags(userList, tags);
+  }
+
+  _filterUsersWithAllTags(userList, tags) {
+    let delimiter = ',';
+    return userList.filter(user => {
+      let userTagTokens = user.tags.split(delimiter);
+      return _.difference(tags, userTagTokens).length === 0;
+    });
+  }
+
+  async usersWithAnyTags(tags=[]) {
```

Comment:
What if this and the AllTags version just passed in a "combiner" function or something to a `_filterUsersWithTags` function. While we're at it, I like just passing in delimiter directly since it is so simple.

```js
 _filterUsersWithAllTags(userList, tags, combiner) {
    return userList.filter(user => {
      let userTagTokens = user.tags.split(',');
      return combiner(tags, userTagTokens).length === 0;
  });
} 
```

and then you can have

```js
  async usersWithAllTags(tags=[]) {
    return _filterUsersWithAllTags(await users, tags, _.difference);
  } 

  async usersWithAnyTags(tags=[]) {
    return _filterUsersWithAllTags(await users, tags, _.intersection);
  } 

```